Providers/nxOMSContainers/containers/conf/container.conf

```diff
@@ -0,0 +1,141 @@
+# Fluentd config file for OMS Docker components
+
+# Forward port 25225 for container logs
+<source>
+	type forward
+	port 25225
+	bind 127.0.0.1
+</source>
+
+# Filter container logs
+<filter docker.**>
+	type filter_docker_log
+	log_path "/var/opt/microsoft/omsagent/log/filter_docker_log.txt"
+</filter>
+
+# Container perf
+<source>
+	type oms_omi
+	object_name "Container"
+	interval 15s
+</source>
+
+# Container inventory
+<source>
+	type omi
+	run_interval 60s
+	tag oms.container.containerinventory
+	items [
+		["root/cimv2","Container_ContainerInventory"]
+	]
+</source>
+
+# Image inventory
+<source>
+	type omi
+	run_interval 60s
+	tag oms.container.imageinventory
+	items [
+		["root/cimv2","Container_ImageInventory"]
+	]
+</source>
+
+# Container service log
+<source>
+	type omi
+	run_interval 30s
+	tag oms.container.servicelog
+	items [
+		["root/cimv2","Container_DaemonEvent"]
+	]
+</source>
+
+# Container log
+<source>
+	type omi
+	run_interval 300s
+	tag oms.container.log
+	items [
+		["root/cimv2","Container_ContainerLog"]
+	]
+</source>
+
+# Container host inventory
+<source>
+	type omi
+	run_interval 60s
+	tag oms.api.ContainerNodeInventory
+	items [
+		["root/cimv2","Container_HostInventory"]
+	]
+</source>
+
+# Container process
+<source>
+	type omi
+	run_interval 60s
+	tag oms.api.ContainerProcess
+	items [
+		["root/cimv2","Container_Process"]
+	]
+</source>
+
+#Kubernetes pod inventory
+<source>
+	type kubepodinventory
+	tag oms.api.KubePodInventory
+	run_interval 60s
+</source>
+
+#Kubernetes events
+<source>
+	type kubeevents
+	tag oms.api.KubeEvents
+	run_interval 60s
+</source>
+
+# Filter for correct format to endpoint
+<filter oms.container.**>
+	type filter_container
+</filter>
+
+<match oms.api.KubePodInventory** oms.api.KubeEvents**>
+	type out_oms_api
+	log_level debug
+	buffer_chunk_limit 5m
+	buffer_type file
+	buffer_path /var/opt/microsoft/omsagent/state/out_oms_api_kubernetes*.buffer
```

Comment:
Why haven't you made this line workspace-specific while other lines referring to the state directory in this file are workspace-specific?